index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header>
+      <div class="nav-wrapper">
+        <nav class="navbar">
+          <div class="navtop">
+            <div class="time">
+              <h2>9:41</h2>
+            </div>
+            <div class="icons">
+              <i class="fa fa-signal" aria-hidden="true"></i>
+              <i class="fa fa-wifi" aria-hidden="true"></i>
+              <i class="fa fa-battery-full" aria-hidden="true"></i>
+            </div>
+          </div>
+          <div class="navbottom">
+            <div class="logo">
+              <h2>Carlyle Codes</h2>
+            </div>
+            <div class="menu">
+              <i class="fas fa-bars"></i>
+            </div>
+          </div>
+        </nav>
+      </div>
+      <div class="content-wrapper">
+        <div class="content">
+          <h1>Hey There. I'm Carlyle</h1>
+          <h3>I am a Software Developer</h3>
+          <h5>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h5>
+        </div>
+        <div class="social-links">
+          <img id="one" class="author" src="images/Vector@2x.png" alt="">
+          <i id="two" class="fab fa-linkedin-in"></i>
+          <img id="three" class="author" src="images/Vector.png" alt="two fingers, peace sign">
+          <img id="four" class="author" src="images/twitter.png" alt="twitter icon">
+          <img id="five" class="author" src="images/m-icon.png" alt="M">
+        </div>
+      </div>
```

Comment:
Following HTML & CSS best practices, it would be nice to use the `section` tag instead of `div` tag for the headline section to improve our structure and semantics. The portfolio will have many sections in the future.